CONTRIBUTING.md

```diff
@@ -0,0 +1,120 @@
+## Getting started
+
+* Please report any issues you encounter during operations.
+* Feel free to create a pull request, preferably with a test or five.
+
+## Setting up a development environment
+
+### Requirements
+
+* Clojure 1.11+
+* OpenJDK 17
+* Node.js 18.x
+* Yarn
+
+For testing: Docker Compose
+
+Please refer to the extensive documentation available on the Metabase website: [Guide to writing a Metabase driver](https://www.metabase.com/docs/latest/developers-guide/drivers/start.html)
+
+Materialize driver's code should be inside the main Metabase repository checkout in `modules/drivers/materialize` directory.
+
+Additionally, you need to tweak Metabase `deps.edn` a bit.
```

Comment:
How should I tweak it? Is it what it is available in the following steps?

Reply:
Yep we have it on line 103 with the `echo` command:

https://github.com/MaterializeInc/metabase-materialize-driver/pull/20/files/f1d769519246850ddebd6efe5d4528224f126495#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R103

Reply:
I've removed that line so it's not confusing.